.github/CODEOWNERS

```
@@ -14,3 +14,4 @@
 
 ## Default reviewers for Intellij extension
 /apps/intellij-extension/ @OxyFlax
+/apps/intellij-extension/build.gradle.kts
```

I'm not sure how the codeowners format works, shouldn't it be:
```suggestion
/apps/intellij-extension/build.gradle.kts @AmadeusITGroup/otter_admins
```
?
From what I seem to understand from the examples of the doc, lefting it empty will override the default

---

Good catch!